tutorial_src/gue/prog/guechk.cpp

```diff
@@ -0,0 +1,62 @@
+#include <stdio.h>
+#include <string.h>
+#include "oi.h"
+
+using namespace std;
+
+void endf(const char* msg, int line, int position) {
+    printf("WRONG\nProgram writes to standard output\n");
+    exit(1);
+}
+
+int main(int argc, char **argv) {
+    if (argc != 4) {
+        fprintf(stderr, "Usage: %s in out wzo\n", argv[0]);
+        return 1;
+    }
+
+    // In file is not needed.
```

Comment:
What is not needed?

Reply:
Ponieważ biblioteczka wypluwa nam OK/WA, więc in nie jest nam potrzebny by oszacować punkty

Reply:
Ok, zrozumiałem to jako "W pliku nie jest potrzebne" i myślałem że coś zgubiłeś.

Reply:
IN FILE is not needed. Jak mogę to lepiej sformułować?

Reply:
Input file?
File with input data?

Reply:
Done